tracee-ebpf/tracee/process_tree.go

```diff
@@ -164,11 +164,11 @@ func NewProcessTree() (*ProcessTree, error) {
 			}
 			processStatus, err := parseProcStatus(data, taskStatus)
 			if err != nil {
-				return nil, err
+				continue
```

Comment:
shouldn't we handle the error somehow, maybe we could push it to the error channel or something similar. WDYT?

Reply:
yes i think it possible.
but actually i don't know if there is any error channel or something like that.
if you know send me a reference to the code and ill add it RN (:

Reply:
I think that we should skip the error handling for now. Walking procfs will lead to many issues when the host is overloaded (pids appearing and disappearing as directories, files being created and deleted, files that can't be read, files in binary format, etc etc).

Reply:
Printing errors to stderr is an alternative, but could make the final user confused about what is actually happening (while there isn't a real problem going on).

Reply:
agreed (:
BTW @rafaeldtinoco do you think i should remove the error check when the calling to `NewProcessTree()`  or verify it wont failed the whole Tracee creation ?

Reply:
I agree with @rafaeldtinoco.
In addition, even when there are no errors, some processes may have been created/exited between the time we initialize the process tree to the time we start monitoring `sched_process_xxx` tracepoints. This mismatch should probably be resolved by reiterating procfs after bpf program started, removing/adding processes as required (we may even need to do this more than once, to the point when there are no more processes to add/remove)

Reply:
> agreed (: BTW @rafaeldtinoco do you think i should remove the error check when the calling to `NewProcessTree()` or verify it wont failed the whole Tracee creation ?

I think that we shuold keep this error check as there might be more "severe" errors, such as not being able to open or read /proc